src/redux/index.ts

```diff
@@ -33,7 +32,6 @@ import {
 } from './globalFilterReducers';
 import {
   ADD_QUICKSTARTS_TO_APP,
-  APP_NAV_CLICK,
```

Comment:
This constant has to be removed as well. And I suspect there is still a redux action that references it.